test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
ok, I'll add the "make sure the sub wasn't created" to that other functest and delete the new functest.